docs/content/overview.md

```diff
@@ -33,6 +33,7 @@ Here is an overview of the major features of each cloud storage system.
 | HDFS                         | -                 | R/W     | No               | No              | -         | -        |
 | HiDrive                      | HiDrive ¹²        | R/W     | No               | No              | -         | -        |
 | HTTP                         | -                 | R       | No               | No              | R         | -        |
+| iCloud Drive                 | -                 | R       | No               | No              | -         | -        |
```

Comment:
This needs an entry in the [optional features](https://rclone.org/overview/#optional-features) table too. I'll put that in the CONTRIBUTING docs.